### PR TITLE
Add a notice on Hacktoberfest limits

### PR DESCRIPTION
**Pull request by encukou**
@hroncok, rather than merging, I'm thinking about making `spamtoberfest` the default branch here for a month. Does that sound good?

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,22 @@
+\ :mushroom: Hacktoberfest limits :mushroom:
+============================================
+
+This repository is attracting many `spam pull requests`_ during `Hacktoberfest`_ 2020.
+
+Due to the overwhelming number of PRs, we have limited interaction with this
+repository to existing contributors.
+If you have a pull request, please request a pull the old-fashioned way:
```
Comment (hroncok) [inline README.rst:8]: Or if you want to open an issue?

**Comment by hroncok** [comment]
> I'm thinking about making spamtoberfest the default branch here for a month. Does that sound good?

Sure. Make it so :)

File: README.rst [hunk 2]
```diff
@@ -1,3 +1,22 @@
...
+send the URL of your fork, and the branch name, to `encukou@gmail.com`_.
+I'll merge manually or open a PR on your behalf.
```
Comment (encukou) [inline README.rst:10]:
```suggestion
I'll merge manually or open a PR on your behalf.

If you found an issue, or want to do anything else we're disallowing this month,
please e-mail the same address.
```

**Review by hroncok — APPROVED** [review]
Let's also pin an issue with the same text.

**Comment by encukou** [comment]
October is over for 2020.

**Pull request closed without merging**